### PR TITLE
FF132 Expr/Relnote: CloseWatcher

### PR DESCRIPTION
**Pull request by hamishwillee**
FF132 supports [`CloseWatcher`](https://developer.mozilla.org/en-US/docs/Web/API/CloseWatcher) in https://bugzilla.mozilla.org/show_bug.cgi?id=1888729 behind the pref `dom.closewatcher.enabled`.

This adds a release note and experimental feature entry.

Related docs work can be tracked in #36126

### CODE REVIEW

File: files/en-us/mozilla/firefox/experimental_features/index.md [hunk 1]
```diff
@@ -1377,6 +1377,49 @@ This enables a fetch request to function as an alternative to {{domxref("Navigat
   </tbody>
 </table>
 
+### CloseWatcher Interface
+
+Built-in web components with "open" and "close" semantics, such as modal dialogs and popovers, can be closed using device-native mechanisms.
+For example, on Windows you can close a dialog using the <kbd>Esc</kbd> key.
```
Comment (pepelsbey) [inline files/en-us/mozilla/firefox/experimental_features/index.md:1383]:
```suggestion
For example, on Android you can close a dialog using the back button.
```

I think there might be a better example explaining this behavior: the back button on Android. We more or less got used to Esc closing popups but the back button triggering the close behavior is something new. [See Chrome 120 release notes](https://developer.chrome.com/blog/new-in-chrome-120#close-watcher).

File: files/en-us/mozilla/firefox/releases/132/index.md [hunk 2]
```diff
@@ -94,6 +94,9 @@ These features are newly shipped in Firefox 132 but are disabled by default. To
 
   This enables a fetch request to function as an alternative to {{domxref("Navigator.sendBeacon()")}} when sending analytics at the end of a session, which has some advantages (you can use HTTP methods other than [`POST`](/en-US/docs/Web/HTTP/Methods/POST), customize request properties, and access the server response via the fetch {{jsxref("Promise")}} fulfillment). It is also available in [service workers](/en-US/docs/Web/API/Service_Worker_API). ([Firefox bug 1906952](https://bugzil.la/1906952)).
 
+- **`CloseWatcher`**: <code>dom.closewatcher.enabled</code>.
+  The {{domxref("CloseWatcher")}} interface enables developers to implement components that can be closed using device-native mechanisms, in the same way as built-in components. For example, in Windows you can close a modal dialog using the <kbd>Esc</kbd> key: this interface allows you to similarly close a custom sidebar. ([Firefox bug 1888729](https://bugzil.la/1888729)).
```
Comment (pepelsbey) [inline files/en-us/mozilla/firefox/releases/132/index.md:98]:
```suggestion
  The {{domxref("CloseWatcher")}} interface enables developers to implement components that can be closed using device-native mechanisms, in the same way as built-in components. For example, on Android you can close a dialog using the back button: this interface allows you to similarly close a custom sidebar. ([Firefox bug 1888729](https://bugzil.la/1888729)).
```

**Review by pepelsbey — COMMENTED** [review]
Looks good! I have a small suggestion though ;)

File: files/en-us/mozilla/firefox/experimental_features/index.md (hunk 1 above)
Comment (hamishwillee) [reply files/en-us/mozilla/firefox/experimental_features/index.md:1383]: Sounds good to me thanks!

**Review by pepelsbey — APPROVED** [review]
Nice :) Thanks

**Pull request merged**